starters/blog/src/templates/blog-post.js

```diff
@@ -64,6 +60,17 @@ const BlogPostTemplate = ({ data, location }) => {
   )
 }
 
+export const Head = ({ data }) => {
+  const post = data.markdownRemark
```

Comment:
You could move that destructing into the function arg already

Reply:
@marvinjude why do you mark this as resolved?

Reply:
@LekoArts it's actually updated now but for some reason, Github doesn't add the "outdated" stamp.  Check the "Files 
changed" section

Reply:
No, this file still has the same `Head` function with the `({ data })` prop

Reply:
Ah! my bad. I updated page template instead